<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDPSEuDataList</class>
 <widget class="QDialog" name="TDPSEuDataList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QPushButton" name="BTN_Insert">
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="BTN_Save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="BTN_Modify">
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="Empty">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="BTN_Delete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableView" name="QTable_TDPS"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
